kernelci.org/content/en/docs/instances/chromeos/chromeos_changelog.md

```diff
@@ -41,8 +42,12 @@ Direct links for each supported board in this release are provided here for conv
 
 ### Known divergences
 
+#### src/third-party/kernel/next
+	- Points to the [Mediatek Integration branch](https://gitlab.collabora.com/google/chromeos-kernel/-/tree/for-kernelci).
+	- Currently only used for cherry board builds because upstream support is still WIP.
+
 #### src/third-party/kernel/upstream
-	- Based on latest v6.2 kernel release.
+	- Based on v6.2.7 stable kernel release.
```

Comment:
Is this not going to follow all the `linux-6.2.y` updates?

Reply:
I thought the ChromeOS kernels from Google were being rebased or pulled LTS branches regularly to get all the fixes, and that's what we would be getting as part of the Chromium OS image updates too.  I guess it doesn't really matter too much for now as the fixed kernels are just to test the rest of the setup.  If we started testing ChromeOS kernel branches on chromeos.kernelci.org then it would probably be useful to get the latest from the branches with all the fixes from LTS.